bibernate-core/src/main/java/com/bobocode/petros/bibernate/exceptions/ExceptionMessages.java

```diff
@@ -8,6 +8,8 @@ public enum ExceptionMessages {
     public static final String COLUMNS_METADATA_MESSAGE = "Couldn't fetch columns metadata. Detail: %s";
     public static final String MAPPING_VALIDATION_ERROR_MSG = "Couldn't validate entity mapping. Detail %s ";
     public static final String MAPPING_VIOLATION_MESSAGE = "Found %d mapping violations. Details: %s";
+    public static final String NULL_ENTITY_PERSISTENCE_CONTEXT_SNAPSHOT_EXCEPTION = "Entity that you want to do snapshot must not be null!";
```

Comment:
First of all, it is an exception _message_, so it is supposed to be ended with  _MESSAGE/_MSG.
Message can be simplified -> `"Snapshot entity must not be null!"`, the same is applicable for cache -> `"Cacheable entity must not be null!"`, moreover you can merge these messages in a single one -> `public static final String NULL_ENTITY_PERSISTENCE_CONTEXT_MESSAGE = "%s must not be null!";`

Reply:
Any updates on this comment? it seems nothing has changed